### PR TITLE
Add: Documentation guidelines

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
#### Types of changes
- [x] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

This PR adds new documentation, Documentation best practices. It also adds a mention to this documentation in the [Engineering guidelines](https://github.com/vtexdocs/dev-portal-content/pull/895/files#diff-7e43d1f595bc50fb2840a4a5f2518f91dcc71faeb6e282da079e33cb9b517ccc).

P.s.: when [556 PR](https://github.com/vtexdocs/devportal/pull/556/files) get deploy, I will update the [line 136](https://github.com/vtexdocs/dev-portal-content/pull/895/files#diff-7e43d1f595bc50fb2840a4a5f2518f91dcc71faeb6e282da079e33cb9b517cccR136) to add the right link for the Documentation guidelines doc.

Related PR: https://github.com/vtexdocs/devportal/pull/556

### CODE REVIEW

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
+---
+title: "Documentation best practices"
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-guidelines.md:2]:
```suggestion
title: "Documentation guidelines"
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-engineering-guidelines.md [hunk 2]
```diff
@@ -132,3 +132,5 @@ The app documentation should include:
 - Technical and commercial requirements for the app's operation.
 - Installation and configuration guides.
 - Answers to frequently asked questions.
+
+For more information on the best practices for creating documentation, refer to [Documentation Guideline](https://developers.vtex.com/docs/guides/vtex-io-documentation-documentation-guidelines)
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-engineering-guidelines.md:136]:
```suggestion
For more information on the best practices for creating documentation, refer to [Documentation guidelines](https://developers.vtex.com/docs/guides/vtex-io-documentation-documentation-guidelines).
```

File: navigation.json [hunk 3]
```diff
@@ -10763,6 +10763,13 @@
               "origin": "",
               "type": "markdown",
               "children": []
+            },
+            {
+              "name": "Documentation best practices",
```
Comment (Mariana-Caetano) [inline navigation.json:10768]:
```suggestion
              "name": "Documentation guidelines",
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+
+✅ Google Tag Manager
+
+❌Google Tag Manager v3.5.2.
+
+### Introduction
+
+The introduction should summarize your app’s purpose and what the user will learn with the documentation.
+
+#### Do’s
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-guidelines.md:63]:
```suggestion
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-engineering-guidelines.md [hunk 5]
```diff
@@ -132,3 +132,5 @@ The app documentation should include:
 - Technical and commercial requirements for the app's operation.
 - Installation and configuration guides.
 - Answers to frequently asked questions.
+
+For more information on the best practices for creating documentation, refer to [Documentation guidelines](https://developers.vtex.com/docs/guides/vtex-io-documentation-documentation-guidelines).
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-engineering-guidelines.md:136]:
```suggestion
For more information on the best practices for creating documentation, refer to [Documentation guidelines](/).
```

File: navigation.json [hunk 6]
```diff
@@ -10763,6 +10763,13 @@
               "origin": "",
               "type": "markdown",
               "children": []
+            },
+            {
+              "name": "Documentation guidelines",
+              "slug": "vtex-io-documentation-docs-guidelines",
+              "origin": "",
+              "type": "markdown",
+              "children": []
```
Comment (Mariana-Caetano) [inline navigation.json:10772]:
```suggestion
            }
```

File: navigation.json [hunk 7]
```diff
@@ -10764,6 +10764,7 @@
               "type": "markdown",
               "children": []
             }
+            }
```
Comment (Mariana-Caetano) [inline navigation.json:10767]:
```suggestion
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md [hunk 8]
```diff
@@ -0,0 +1,218 @@
...
+
+*![insert-an-image-preview](/)*
+
+## Configuration
+
+Once you have installed the app, you can {describe the app's configuration in the VTEX Admin, for example}.
+
+1. {first step}.
+2. {second step}.
+3. {third step}.
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md:193]:
```suggestion
1. *Describe the first step here*.
2. *Describe the second step here*.
3. *Describe the third step here*.
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md [hunk 9]
```diff
@@ -0,0 +1,220 @@
...
+> ⛔ Danger message goes here.
+```
+
+## Documentation template
+
+A basic documentation template structure with the previously described sections for you to start.
+
+<details>
+<summary><b>Documentation template</b></summary>
+<br>
```
Comment (Mariana-Caetano) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md:146]:
```suggestion
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+
+✅ Live Shopping
+
+❌ Live Shopping app
+
+---
+
+✅ Google Tag Manager
+
+❌Google Tag Manager v3.5.2.
```
Comment (carolinamenezes) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md:58]:
```suggestion
| ✅ Do | ❌ Don't |
|-------|---------|
| Google Tag Manager | Google Tag Manager v3.5.2.|
| Live Shopping | Live Shopping app |
```
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md [hunk 11]
```diff
@@ -0,0 +1,219 @@
...
+```
+
+## Documentation template
+
+A basic documentation template structure with the previously described sections for you to start.
+
+<details>
+<summary><b>Documentation template</b></summary>
+
+````bash
```
Comment (carolinamenezes) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md:147]:
```suggestion
```bash
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md [hunk 12]
```diff
@@ -0,0 +1,219 @@
...
+
+## Contributors
+
+Thanks go to these wonderful people:
+
+- `{insert the GitHub username}`
+
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome.
+
+````
```
Comment (carolinamenezes) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md:212]:
```suggestion
```
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md (hunk 10 above)
Comment (carolinamenezes) [reply docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md:58]: A visualização em tabela fica interessante para esses exemplos pq dá para ver lado a lado. Sugiro aplicar nos outros exemplos também

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md [hunk 13]
```diff
@@ -0,0 +1,219 @@
...
+```md
+> ⛔ Danger message goes here.
+```
+
+## Documentation template
+
+A basic documentation template structure with the previously described sections for you to start.
+
+<details>
+<summary><b>Documentation template</b></summary>
```
Comment (carolinamenezes) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md:145]:
```suggestion
<summary>Documentation template</summary>
```

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-engineering-guidelines.md [hunk 14]
```diff
@@ -132,3 +132,5 @@ The app documentation should include:
 - Technical and commercial requirements for the app's operation.
 - Installation and configuration guides.
 - Answers to frequently asked questions.
+
+For more information on the best practices for creating documentation, refer to [Documentation guidelines](/).
```
Comment (carolinamenezes) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-engineering-guidelines.md:136]: Ajustar link

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md [hunk 15]
```diff
@@ -0,0 +1,219 @@
...
+
+````bash
+# {Insert the app's name}
+
+The `{insert app's name}` is responsible for `{app's purpose}` so you can `{job to be done}`.
+
+    `![insert-an-image-preview](/)`
```
Comment (carolinamenezes) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md:152]:
```suggestion
    ![insert-an-image-preview](/)
```

**Review by carolinamenezes — CHANGES_REQUESTED** [review]

File: docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md [hunk 16]
```diff
@@ -0,0 +1,204 @@
...
+```
+
+## Documentation template
+
+A basic documentation template structure with the previously described sections for you to start.
+
+<details>
+<summary><b>Documentation template</b></summary>
+
+```bash
```
Comment (carolinamenezes) [inline docs/vtex-io/Perks/vtex-io-documentation-homologation-requirements-for-vtex-app-store/vtex-io-documentation-docs-guidelines.md:132]:
```suggestion
```markdown
```

**Review by carolinamenezes — APPROVED** [review]

**Pull request merged**